static/app/views/issueDetails/groupPriority.tsx

```diff
@@ -36,11 +41,20 @@ function GroupPriority({group}: GroupDetailsPriorityProps) {
       api,
       {
         orgId: organization.slug,
-        projectId: group.project.slug,
         itemIds: [group.id],
         data: {priority},
+        failSilently: true,
```

Comment:
By default this function has its own error indicator, but we need to do it manually so that we can clear the loading indicator first.